### PR TITLE
<FEAT> Duration Estimation Method Swap

### PR DESCRIPTION
**Pull request by SEAJang**
# Pull Request Key Information

## Reason for change

Duration estimation was initially hard coded. 

## What was changed

Users are now able to set, per program, a duration estimation method to calculate emissions. 

## Intended Purpose

Allow users to choose a duration estimation method and create a framework for future methods to be incorporated easily.

## Testing Completed

Manual testing - ran with the test simulation where P_Air was using the newly defined component-based estimation, while P_OGI was using the measurement-based-conservative method. 
[SampleOutputFiles.zip](https://github.com/LDAR-Sim/LDAR_Sim/files/15101417/SampleOutputFiles.zip)

All unit tests passed. 
[result.txt](https://github.com/LDAR-Sim/LDAR_Sim/files/15101386/result.txt)

### CODE REVIEW

File: LDAR_Sim/src/file_processing/output_processing/summary_outputs.py [hunk 1]
```diff
@@ -99,4 +99,63 @@ def generate_emissions_summary(directory: str, outputs_mapper: SummaryOutputMapp
...
+            output_file_constants.SummaryFileColumns.CommonColumns.SIMULATION_NUMBER,
+        ],
+        how="outer",
+    )
+    # Fill NaN values with zeros
+    merged.fillna(0, inplace=True)
+    return merged
+
+
+def generate_emissions_estimation_summary(directory: str, outputs_mapper: SummaryOutputMapper):
```
Comment (ThomasGalesloot) [inline LDAR_Sim/src/file_processing/output_processing/summary_outputs.py:117]: Design Question: What is the point of making these just to basically drop all the columns?

**Review by ThomasGalesloot — CHANGES_REQUESTED** [review]
Great Work overall - I've asked some questions and noted a few nitpicks, but the biggest thing is that new tests are needed for calling summarize_program_outputs with the 2 new mappings.

File: LDAR_Sim/src/constants/output_file_constants.py [hunk 2]
```diff
@@ -172,11 +169,58 @@ class EMIS_DATA_COL_ACCESSORS:
...
     RECORDED = "Recorded"
     RECORDED_BY = "Recorded By"
     REPAIRABLE = "Repairable"
+    SURVEY_LEVEL = "Survey Level"
+    SURVEY_START_DATE = "Survey Start Date"
+    SURVEY_COMPLETION_DATE = "Survey Completion Date"
+    METHOD = "Method"
+    START_DATE = "Start Date"
+    END_DATE = "End Date"
+    PREV_CONDITION = "Use Previous Condition"
```
[the remark this answers is not in the document]
Comment (SEAJang) [reply LDAR_Sim/src/constants/output_file_constants.py:183]: Some of them are column headers that are saved.

**Comment by SEAJang** [comment]
[results.txt](https://github.com/LDAR-Sim/LDAR_Sim/files/15145375/results.txt)

File: LDAR_Sim/src/file_processing/output_processing/summary_outputs.py (hunk 1 above)
Comment (SEAJang) [reply LDAR_Sim/src/file_processing/output_processing/summary_outputs.py:117]: Which columns in particular is this comment referring to?

**Review by ThomasGalesloot — APPROVED** [review]

**Pull request merged**